jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/CheckpointManager.scala

```diff
@@ -127,7 +133,7 @@ private[spark] class CheckpointManager(sc: SparkContext, checkpointPath: String)
 
 object CheckpointManager {
 
-  private[spark] def extractParams(params: Map[String, Any]): (String, Int) = {
+  private[spark] def extractParams(params: Map[String, Any]): (String, Int, Boolean) = {
```

Comment:
I think it may be a time to create a case class

case class Params(checkpointPath: String, checkpointInverval: Int, skip_clean_checkpoint: Boolean)

Reply:
done